src/styles/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  width: ($day-size + $gap) * $colums;
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__day {
+    position: relative;
+    box-sizing: border-box;
+
+    height: $day-size;
+    width: $day-size;
+
+    background-color: #eee;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)